lib/ReactViews/Map/Panels/SharePanel/Print/PrintView.tsx

```diff
@@ -152,7 +148,15 @@ const PrintView = (props: Props) => {
     canShorten(props.terria)
       ? buildShortShareLink(props.terria, props.viewState, {
           includeStories: false
-        }).then(setShareLink)
+        })
+          ?.then(url => {
```

Comment:
I'm not keen on `?.then`. Why are there 3 states that this function can return? Is it super important that `undefined` is synchronous?

Reply:
Sorry not sure why those were there. Removed